Dockerfile-Watchman

```diff
@@ -0,0 +1,35 @@
+FROM python:3.7.5 as builder
```

Comment:
This whole file is gone now, right? Probably some rebasing leftover.

Reply:
Yes, this should be removed, I have probably added it with a mistake.